HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -433,6 +438,49 @@
+    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Zones/h:Zone/h:Spaces/h:Space'>
+      <sch:assert role='ERROR' test='count(h:FloorArea) = 1'>Expected 1 element(s) for xpath: FloorArea</sch:assert>
+    </sch:rule>
+  </sch:pattern>
+
+  <sch:pattern>
+    <sch:title>[Space=Conditioned]</sch:title>
+    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Zones/h:Zone[h:ZoneType="conditioned"]/h:Spaces/h:Space'>
+      <sch:assert role='ERROR' test='count(h:extension/h:ManualJInputs/h:InternalLoadsSensible) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/ManualJInputs/InternalLoadsSensible</sch:assert>
+      <sch:assert role='ERROR' test='number(h:extension/h:ManualJInputs/h:InternalLoadsSensible) &gt;= 0 or not(h:extension/h:ManualJInputs/h:InternalLoadsSensible)'>Expected extension/ManualJInputs/InternalLoadsSensible to be greater than or equal to 0</sch:assert>
+      <sch:assert role='ERROR' test='count(../*/h:extension/h:ManualJInputs/h:InternalLoadsSensible) = count(../h:Space) or count(../*/h:extension/h:ManualJInputs/h:InternalLoadsSensible) = 0'>Expected extension/ManualJInputs/InternalLoadsSensible for all Spaces or no Spaces</sch:assert>
+      <sch:assert role='ERROR' test='count(h:extension/h:ManualJInputs/h:NumberofOccupants) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/ManualJInputs/NumberofOccupants</sch:assert>
+      <sch:assert role='ERROR' test='number(h:extension/h:ManualJInputs/h:NumberofOccupants) &gt;= 0 or not(h:extension/h:ManualJInputs/h:NumberofOccupants)'>Expected extension/ManualJInputs/NumberofOccupants to be greater than or equal to 0</sch:assert>
+      <sch:assert role='ERROR' test='count(../*/h:extension/h:ManualJInputs/h:NumberofOccupants) = count(../h:Space) or count(../*/h:extension/h:ManualJInputs/h:NumberofOccupants) = 0'>Expected extension/ManualJInputs/NumberofOccupants for all Spaces or no Spaces</sch:assert>
+      <sch:assert role='ERROR' test='count(h:extension/h:ManualJInputs/h:FenestrationLoadProcedure) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/ManualJInputs/FenestrationLoadProcedure</sch:assert>
+      <sch:assert role='ERROR' test='h:extension/h:ManualJInputs/h:FenestrationLoadProcedure[text()="standard" or text()="peak"] or not(h:extension/h:ManualJInputs/h:FenestrationLoadProcedure)'>Expected extension/ManualJInputs/FenestrationLoadProcedure to be 'standard' or 'peak'</sch:assert>
+    </sch:rule>
+  </sch:pattern>
+
+  <sch:pattern>
+    <sch:title>[AttachedToSpace]</sch:title>
+    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Enclosure/*/*/h:AttachedToSpace'>
+      <sch:assert role='ERROR' test='count(../../../../h:Systems/h:HVAC/h:HVACPlant/*/h:AttachedToZone) = (count(../../../../h:Systems/h:HVAC/h:HVACPlant/h:HeatingSystem) + count(../../../../h:Systems/h:HVAC/h:HVACPlant/h:CoolingSystem) + count(../../../../h:Systems/h:HVAC/h:HVACPlant/h:HeatPump))'>Expected AttachedToZone for each HVAC system</sch:assert>
```

Comment:
Nice check. Though as your test shows, this currently triggers on every AttachedToSpace element. I think you could instead trigger it once for the building by doing something like this (note: I haven't tested it):
```suggestion
    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails[h:Enclosure/*/*/h:AttachedToSpace]'>
      <sch:assert role='ERROR' test='count(h:Systems/h:HVAC/h:HVACPlant/*/h:AttachedToZone) = (count(h:Systems/h:HVAC/h:HVACPlant/h:HeatingSystem) + count(../../../../h:Systems/h:HVAC/h:HVACPlant/h:CoolingSystem) + count(h:Systems/h:HVAC/h:HVACPlant/h:HeatPump))'>Expected AttachedToZone for each HVAC system</sch:assert>
```